main.js

```diff
@@ -1,3 +1,3 @@
-import add from './add.js'
+import add from './add.js';
```

Comment:
This one emits an error:
```
message: 'Unexpected use of file extension "js" for "./add.js" (import/extensions)'
source: 'eslint'
code: 'import/extensions'
```

When the extension is removed, the server cant find the files because the extension is missing.

Reply:
Yeah exactly. I think the spec requires the extension? I think we should check and follow the spec.